.github/workflows/build-test.yml

```diff
@@ -13,7 +13,10 @@ permissions:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-14, openeuler-6cpu-14gb]
```

Comment:
It looks like `openeuler-6cpu-14gb` is not one of the runners we have access to. Where did you find it?

Reply:
This is the name of a runner set provided by the CNCF Infra team.
I have pinged @jeefy and @krook by email.

Reply:
I let CI run and, as of this comment, the `openeuler-6cpu-14gb` one is queued but no runner has picked it up.

Reply:
It appears there is a connectivity problem with the Linux aarch64 VMs. We are working on resolving it! Probably early next week!

Reply:
Looks like it's still not ready. I'll try kicking it again next week.

Reply:
This is just waiting on a configuration update from @jeefy who's traveling this week. Should be back up soon!